app/views/quotes/_quote.html.erb

```diff
@@ -1 +1,7 @@
-<div><%= quote.author %> said, <q><%= quote.body %></q>, in the <%= quote.location %></div>
+<div>
+  <%= quote.author %> said, <q><%= quote.body %></q>, in the <%= quote.location %>
+  <% if quote.user == current_user %>
```

Comment:
This is still a concern.  Would like some more input before making changes possibly.  This kind of control I implemented is just a temporary measure really until we decide either yes we want more functionality or we just don't even want this base amoutn.